JavaSnackBar/src/snackBar/Snack.java

```diff
@@ -0,0 +1,88 @@
+package snackBar;
+ 
+public class Snack
+{
+	// fields
+	private static int maxId = 0;
+	private int id;
+	private String name;
+	private int quantity;
+	private double cost;
+	private int vendingMachineId;
+
+	// constructor
+	public Snack(String name, int quantity, double cost, int vendingMachineId)
+	{
+		maxId++;
+
+		id = maxId;
+
+		this.name = name;
+		this.quantity = quantity;
+		this.cost = cost;
+		this.vendingMachineId = vendingMachineId;
+	}
+
+	// methods
+	public int getId()
+	{
+		return id;
+	}
+
+	public void setName(String name)
+	{
+		this.name = name;
+	}
+
+	public String getName()
+	{
+		return name;
+	}
+
+	public void setQuantity(int quantity)
+	{
+		this.quantity = quantity;
+	}
+
+	public int getQuantity()
+	{
+		return quantity;
+	}
+
+	public void setCost(double cost)
+	{
+		this.cost = cost;
+	}
+
+	public double getCost()
+	{
+		return cost;
+	}
+
+	public void addQuantity(int newQuantity)
+    {
+        quantity = quantity + newQuantity;
+    }
+
+	public void setVendingMachineId(int vendingMachineId)
+	{
+		this.vendingMachineId = vendingMachineId;
+	}
+
+	public int getVendingMachineId()
+	{
+		return vendingMachineId;
+	}
+
+	// Object
+	@Override
+	public String toString()
+	{
+		String rtnString = "id: " + id + "\n" +
+							"name: " + name + "\n" +
+							"quantity: " + quantity + "\n" +
+							"cost: " + cost + "\n" +
+							"vendingMachineId: " + vendingMachineId + "\n";
+		return rtnString;
+	}
+}
```

Comment:
I see you are overriding the toString method here but haven't used it yet. If you wanted to use it, you could pop over to the Main.java file and just use 
` System.out.println(s1..toString());
`